### PR TITLE
Added structure and conf.py changes

### PR DESCRIPTION
**Pull request by Souptacular**
set-up structure and made conf.py changes for Solidity syntax highlighting.

Still need to create the rest of the .rst pages and obtain .md pages from frontier guide (they aren't available publicly afaik).

### CODE REVIEW

File: source/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+###################
+Introduction
+###################
+
+.. toctree::
+   :maxdepth: 2
+
+   what-is-ethereum.rst
+   the-homestead-release.rst
+   solidity-in-depth.rst
```
Comment (zelig) [inline source/introduction.rst]: this should not be here right?

Comment (Souptacular) [inline source/introduction.rst]: I don't see why not. the toctree is a common thing for the the larger sub-pages of a guide. It won't be the *only* thing necessarily, but I was putting it there at first so there wouldn't be a blank page.

Comment (zelig) [inline source/introduction.rst]: no i meant solidity-in-depth is not supposed to be in the intro

File: source/index.rst [hunk 2]
```diff
@@ -20,3 +31,107 @@ Indices and tables
...
+  - Backup/Import Accounts
+  	* Backing Up Wallets
+	* Importing Wallets [including pre-sale wallets]
+	* Online wallets, Paper Wallets and Cold Storage
+  - Ether
+    * What is Ether?
+    * [Table of Sub Units]
+    * Getting Ether [Will point to mining and exchanges pages with explanation of both]
+    * Sending Ether
+  - Mining
```
Comment (zelig) [inline source/index.rst]: so here we need a section on running a node, syncing, fast syncing and light client
network connectivity , NAT, discovery and peers

File: source/index.rst [hunk 3]
```diff
@@ -20,3 +31,107 @@ Indices and tables
...
+    * Javascript
+    * [Link to blockchain explorers page]
+
+
+Developing on Ethereum
+  - Developer Tools
+    * IDEs/Frameworks
+    * High Level Languages
+    * Other Tools
+  - DApp Development
```
Comment (zelig) [inline source/index.rst]: solidity resources should come here right

**Comment by Souptacular** [comment]
Fixed issues and added structure of documents and placeholder pages/sections for the documentation. Can be viewed on https://ethereum-homestead.readthedocs.org/en/latest/ or built locally (I fixed the issue preventing local builds).

**Comment by zelig** [comment]
fantastic thx

**Comment by zelig** [comment]
merged :+1:

**Pull request merged**